<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>1024</width><height>768</height></rect>
  </property>
  <property name="windowTitle"><string>Сервер мессенджера</string></property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex"><number>0</number></property>
      <widget class="QWidget" name="messagesTab">
       <attribute name="title"><string>Сообщения</string></attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="messagesView"/> <!-- Проверьте это имя -->
        </item>
        <item>
         <widget class="QPushButton" name="refreshMessagesButton">
          <property name="text"><string>Обновить</string></property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="usersTab">
       <attribute name="title"><string>Пользователи</string></attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="usersView"/> <!-- Проверьте это имя -->
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="banButton"> <!-- Проверьте это имя -->
            <property name="text"><string>Забанить</string></property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="kickButton"> <!-- Проверьте это имя -->
            <property name="text"><string>Отключить</string></property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshUsersButton"> <!-- Проверьте это имя -->
            <property name="text"><string>Обновить</string></property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
